src/go/pt-k8s-debug-collector/dumper/dumper.go

```diff
@@ -113,8 +122,36 @@ func New(location, namespace, resource string, kubeconfig string, forwardport st
 			"perconaservermongodbs",
 		)
 	}
+	sslSecrets := make([]sslSecret, 0)
 	filePaths := make([]string, 0)
-	if resourceType(resource) == "pxc" {
+	switch resourceType(resource) {
+	case "pg":
+		sslSecrets = append(sslSecrets,
+			sslSecret{
+				secret:    "{{ .Name }}-ssl-ca",
+				resource:  "perconapgclusters",
```

Comment:
here and in https://github.com/percona/percona-toolkit/blob/9b3d92997a32395c7082b5990e363275c60c1711/src/go/pt-k8s-debug-collector/dumper/dumper.go#L85 it's better to use a resource type name with percona's domain. If another operator with same short name is installed (or v1 and v2 ) it causes conflicts
```
-                       "perconapgclusters",
-                       "pgclusters",
-                       "pgpolicies",
-                       "pgreplicas",
-                       "pgtasks",
+                       "perconapgclusters.pg.percona.com",
+                       "pgclusters.pg.percona.com",
+                       "pgpolicies.pg.percona.com",
+                       "pgreplicas.pg.percona.com",
+                       "pgtasks.pg.percona.com",
                )
        case "pgv2":
                resources = append(resources,
-                       "perconapgbackups",
-                       "perconapgclusters",
-                       "perconapgrestores",
+                       "perconapgbackups.pgv2.percona.com",
+                       "perconapgclusters.pgv2.percona.com",
+                       "perconapgrestores.pgv2.percona.com",
```